src/modules/Reviews/ReviewList/ReviewsList.tsx

```diff
@@ -38,8 +38,8 @@ const cards: CardProps[] = [
 
 const ReviewList = () => {
   const renderCards = () =>
-    cards.map(({ imageUrl, name, jobName, text }) => (
-      <Card imageUrl={imageUrl} name={name} jobName={jobName} text={text} />
+    cards.map(({ imageUrl, name, jobName, text }, index) => (
```

Comment:
put the cards render logic inside the returned component tree, not as external function